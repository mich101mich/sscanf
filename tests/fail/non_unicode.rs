fn main() {
    type AyÌ†yÌ†yÌ† = u8;
    sscanf::scanf!("Hi", "yÌ†ğŸ˜›yÌ†{AyÌ†yÌ†yÌ†:yÌ†}yÌ†ğŸ˜›yÌ†");
}