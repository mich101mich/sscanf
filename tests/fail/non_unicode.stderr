error: unrecognized format option.
       Hint: Regex format options must start and end with '/':
       At "yÌ†ğŸ˜›yÌ†{AyÌ†yÌ†yÌ†:yÌ†}yÌ†ğŸ˜›yÌ†"
                     ^
 --> tests/fail/non_unicode.rs:3:26
  |
3 |     sscanf::scanf!("Hi", "yÌ†ğŸ˜›yÌ†{AyÌ†yÌ†yÌ†:yÌ†}yÌ†ğŸ˜›yÌ†");
  |                          ^^^^^^^^^^^^^^^^^^
